packages/components/src/components/primitives/Tooltip/Tooltip.tsx

```diff
@@ -16,30 +24,50 @@ type Props = {
   tooltip: React.ReactNode;
   element: React.ReactNode;
   testId?: string;
-  backgroundColor?: string;
+  backgroundColor?: BackgroundColor;
+  className?: string;
 };
 
+const TooltipWrapper = styled(ReactTooltip)`
+  &:after {
+    ${({ background }) =>
+      background &&
+      (background === BackgroundColor.Primary100
+        ? tw`border-fill-primary-100`
+        : tw`border-fill-primary-75`)}
```

Comment:
Could we set `Primary100` or `Primary75` as default value for `backgroundColor` and skip this condition and just assign to background whatever `backgroundColor` prop is received?